### PR TITLE
core/video_core: Fix a few warnings when compiling on MSVC.

### PR DESCRIPTION
**Pull request by lioncash**

### CODE REVIEW

**Comment by archshift** [comment]
LGTM

File: src/core/arm/skyeye_common/vfp/vfpsingle.cpp [hunk 1]
```diff
@@ -388,7 +388,7 @@ static u32 vfp_single_fsqrt(ARMul_State* state, int sd, int unused, s32 m, u32 f
         } else {
             u64 term;
             s64 rem;
-            vsm.significand <<= !(vsm.exponent & 1);
+            vsm.significand <<= static_cast<u32>((vsm.exponent & 1) == 0);
```
Comment (yuriks) [inline src/core/arm/skyeye_common/vfp/vfpsingle.cpp]: Why the cast?

Comment (lioncash) [inline src/core/arm/skyeye_common/vfp/vfpsingle.cpp]: Because it causes a warning otherwise (using a bool in an arithmetic expression directly)

**Comment by Subv** [comment]
Why is this PR named "core/video_core" ?

**Comment by archshift** [comment]
Because it involves changes to core and video_core...

**Comment by bunnei** [comment]
LGTM

**Pull request merged**